test/Microsoft.Net.Http.Headers.Tests/DateParserTest.cs

```diff
@@ -2,29 +2,41 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.Collections.Generic;
 using Xunit;
 
 namespace Microsoft.Net.Http.Headers
 {
     public class DateParserTest
     {
-        [Fact]
-        public void TryParse_SetOfValidValueStrings_ParsedCorrectly()
+        [Theory, MemberData(nameof(ValidStringData))]
```

Comment:
Per my comment in some other PR, split the attributes into two lines, each like `[Foo]`.